karma.conf.js

```diff
@@ -0,0 +1,63 @@
+module.exports = function (config) {
+    config.set({
+        
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: './',
+        
+        // frameworks to use
+        // some available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+        frameworks: ['jasmine'],
+        
+        // list of files / patterns to load in the browser
+        files: [
+            './node_modules/angular/angular.js',
+            './node_modules/angular-mocks/angular-mocks.js',
+
+            './src/javascript/app/ng-wig/ng-wig.js',
+            './src/javascript/app/**/!(ng-wig|app).js'
+        ],
+        
+        proxies: {
+            '/': 'http://localhost:8888/'
+        },
+
+        // preprocess matching files before serving them to the browser
+        // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+        preprocessors: {
+            'src/javascript/app/**/!(*spec).js': 'coverage'
+        },
+        
+        // test results reporter to use
+        // possible values: 'dots', 'progress', 'coverage'
+        // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+        reporters: ['progress', 'coverage'],
+        
+        coverageReporter: {
+            type: 'lcov',
+            dir: 'test/coverage'
+        },
+
+        // web server port
+        port: 9876,
+        
+        // enable / disable colors in the output (reporters and logs)
+        colors: true,
+        
+        // level of logging
+        // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+        logLevel: config.LOG_INFO,
+        
+        // enable / disable watching file and executing tests whenever any file changes
+        autoWatch: true,
```

Comment:
I think for one time test we can put 

    autoWatch: false,
    singleRun: true